scripts/upgrade/TimeConstraints.sol

```diff
@@ -47,22 +55,44 @@ contract TimeConstraints {
         }
     }
 
+    /// @notice Checks that the transaction can only be executed within a specific time range during the day and emits an event.
+    /// @param startDayTime The start time of the allowed range in seconds since midnight (UTC).
+    /// @param endDayTime The end time of the allowed range in seconds since midnight (UTC).
+    function checkTimeWithinDayTimeAndEmit(Duration startDayTime, Duration endDayTime) external {
+        checkTimeWithinDayTime(startDayTime, endDayTime);
+        emit PerformedWithinDayTime(startDayTime, endDayTime);
+    }
+
     /// @notice Checks that the transaction can only be executed after a specific timestamp.
     /// @param timestamp The Unix timestamp after which the function can be executed.
-    function checkExecuteAfterTimestamp(Timestamp timestamp) external view {
+    function checkTimeAfterTimestamp(Timestamp timestamp) public view {
```

Comment:
This method currently checks whether the current time is greater than or equal to the timestamp, which doesn’t match the method name. Let’s update the condition in the if statement to align with the method’s intended behavior.

Reply:
Same in the `checkTimeBeforeTimestamp()` method